sdk/ml/azure-ai-ml/tests/environment/unittests/test_environment_operations_registry.py

```diff
@@ -0,0 +1,60 @@
+from unittest.mock import Mock, patch
+
+import pytest
+
+from azure.ai.ml._restclient.v2022_05_01.models import (
+    EnvironmentVersionData,
+    EnvironmentVersionDetails,
+)
+
+from azure.ai.ml._scope_dependent_operations import OperationConfig, OperationScope
+from azure.ai.ml.operations import EnvironmentOperations
+from azure.ai.ml import load_environment
```

Comment:
Seems that this import is unused

Reply:
since these are tests, I will take care of this in the next PR